x-pack/test/functional/page_objects/rollup_page.js

```diff
@@ -39,15 +84,20 @@ export function RollupPageProvider({ getService, getPageObjects }) {
       await testSubjects.setValue('rollupIndexName', name);
     }
 
-    async moveToNextStep() {
+    async moveToNextStep(stepNum) {
       await testSubjects.click('rollupJobNextButton');
+      return stepNum + 1;
     }
 
     async setJobInterval(time) {
       await testSubjects.setValue('rollupJobInterval', time);
     }
 
-    async saveJob() {
+    async saveJob(startImmediately) {
+      if (startImmediately) {
+        const checkbox = await find.byCssSelector('.euiCheckbox');
```

Comment:
Nit: This could break if we add another checkbox to the UI. Can we use a test subject selector?

Reply:
Needs to click on the div of the checkbox because of the div intercepting the click. I will research a good way to do this and create an issue for it.